### DISCUSSION TITLE
Issue UI: gradio/glass.css not found / and UI locked until page reload

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#2854] [Q&A]
Not a big issue, just a headsup:

| sd | INFO | ui | Settings: changed=1 ['gradio_theme']
| sd | ERROR | middleware | API error: GET: http://127.0.0.1:7860/file=javascript/gradio/glass.css?1707758300.0394342 {'error': 'HTTPException', 'code': 404, 'detail': 'File not found: javascript/gradio/glass.css.', 'body': '', 'errors': '404: File not found: javascript/gradio/glass.css.'}

Once chosing this theme and getting this error, it is not possible anymore, to select another theme. The UI sticks to the last working one. Solution is to reload SD.Next.

**Comment by vladmandic** [comment]
fixed in latest dev branch.

**Reply by ameysh** [reply]
I think this issue has come back.

Steps to reproduce:
1. Set theme type to "None"
2. Change the UI theme to any from the list (I have tried 10+ themes from the list and get the same error)

> App version: 0d7c025a Windows, last updated 30 June 2025

Error:
API error: GET: http://127.0.0.1:7860/file=javascript/gradio/glass.css?1751244766.8013687
                         {'error': 'HTTPException', 'code': 404, 'detail': 'File not found:
                         javascript/gradio/glass.css.', 'body': '', 'errors': '404: File not found:
                         javascript/gradio/glass.css.'}
                         
                      
Please let me know if I can provide any more details to help you.

**Reply by vladmandic** [reply]
there was a bug where theme-type=None was not recognized, that will be fixed shortly.
but in general, Standard and ModernUI are maintained and supported.
theme type none is there for people that want to experiment creating their ui from scratch, its not intended to be used as normal ui and will look pretty broken by default.

btw, one thing i'd ask is not to comment on topics that are closed for over a year, no matter how similar they appear. if last comment is over a year old, please create a new issue.

**Reply by ameysh** [reply]
Thank you for the upcoming bugfix in advance, and the explanation about theme types! My apologies for commenting on an old issue. I was trying not to be redundant by opening another one for the exact same error. I will keep this in mind going forward!
